conf/application.props.example

```diff
@@ -0,0 +1,19 @@
+# Applications
+# ------------
+# This section is used to configure Fluo applications which are indentified by a unique application 
+# name (which replaces <app> below).  Each application has the following configuration settings:
+#
+# <app>.repo                # Git repo containing code for application. It is cloned when the app is run.
+# <app>.branch              # Branch to checkout in git repo
+# <app>.command.pre.init    # Command in application repo to run before initializing and starting Fluo.
+#                           # FLUO_HOME & FLUO_APP_NAME will be set in env when run.  This command
+#                           # can do things like change fluo config and copy observers
+# <app>.command.post.start  # Script to run from application repo after Fluo is started
+#
+# Applications can be run using the following command 'fluo-dev run <app>`.  Below is configuration for
+# a 'stress' application which can be run using 'fluo-dev run stress':
+
+stress.repo=https://github.com/fluo-io/fluo-stress
```

Comment:
I wanted to run a test from a local git repo containg stress (with a local branch).   I changed this to `file:///<location of local stress repo>` and it worked great.   May be worth commenting on this, since using fluo-dev to test local changes is really nice.